employee-rostering-frontend/src/ui/pages/shift/__snapshots__/EditShiftModal.test.tsx.snap

```diff
@@ -122,11 +122,11 @@ exports[`Edit Shift Modal should render correctly when closed 1`] = `
     </InputGroup>
     <InputGroup>
       <Label>
-        Spot
+        Trans(i18nKey=spot)
       </Label>
       <TypeaheadSelectInput
         aria-label="Spot"
```

Comment:
Shouldn't you also use translation for the aria-label's (applies to many inputs in this file)?

Reply:
Probably (although 99% tests use aria-labels to locate elements to click, so this is a major change for a separate PR)